.github/workflows/_test.yml

```diff
@@ -60,3 +60,32 @@ jobs:
       uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # 3.1.4
       env:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  test-windows-macos:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    name: Go unit tests (${{ matrix.os }})
+    steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Setup go
+      uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+      with:
+        go-version-file: ./go.mod
+
+    - uses: actions/cache@e12d46a63a90f2fae62d114769bbf2a179198b5c # 3.3.3
+      with:
+        path: |
+          ~/.cache/go-build
+          ~/go/pkg
+        key: golangci-lint.cache-{platform-arch}-{interval_number}-{go.mod_hash}
+        restore-keys: |
+          golangci-lint.cache-{interval_number}-
+          golangci-lint.cache-
+        env:
+          CGO_ENABLED=0
+    - run: make unittest GO_TEST_FLAGS="-tags test_sqlite_only"
```

Comment:
We probably need the `CGO_ENABLED=0` here and in the step below this onw.

Reply:
you can add:

```yaml
env:
          CGO_ENABLED=0
```

as a global environmnet before the workflow steps:

https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow